File: packages/jsii-pacmak/test/expected.java/org/jsii/tests/calculator/Thrower.java
package org.jsii.tests.calculator;
@org.jsii.Jsii(module = org.jsii.tests.calculator.$Module.class, fqn = "jsii$jsii_calc$.Thrower")
public class Thrower extends org.jsii.JsiiObject {
    protected Thrower(final org.jsii.JsiiObject.InitializationMode mode) {
        super(mode);
    }
    public Thrower() {
        super(org.jsii.JsiiObject.InitializationMode.Jsii);
        org.jsii.JsiiEngine.getInstance().createNewObject(this);
    }
    public void throwError() {
        this.jsiiCall("throwError", Void.class);
    }
}

File: packages/jsii-pacmak/test/expected.dotnet/AWS.Jsii.Tests.Calculator/Thrower.cs
using AWS.Jsii.Runtime.Deputy;

namespace AWS.Jsii.Tests.Calculator
{
    [JsiiClass("jsii-calc", "jsii$jsii_calc$.Thrower", "[]")]
    public class Thrower : DeputyBase
    {
        public Thrower(): base(new DeputyProps(new object[]{}))
        {
        }

        protected Thrower(ByRefValue reference): base(reference)
        {
        }

        protected Thrower(DeputyProps props): base(props)
        {
        }

        [JsiiMethod("throwError", null, "[]")]
        public virtual void ThrowError()
        {
            InvokeVoidMethod(new object[]{});
        }
    }
}